tsconfig.json

```diff
@@ -17,9 +17,10 @@
     "noUnusedParameters": false,
     "noUnusedLocals": false,
     "forceConsistentCasingInFileNames": true,
-    "lib": ["es2017", "esnext.asynciterable"],
+    "lib": ["es2020", "esnext.asynciterable"],
     "types": ["node", "jest"],
-    "baseUrl": "."
+    "baseUrl": ".",
+    "skipLibCheck": true
```

Comment:
There's still a bunch of very old dependencies that have broken types now. Will remove this after they have all been updated.